lightController/lightController.ino

```diff
@@ -144,6 +154,71 @@ void draw_rainbow()
     }
 }
 
+
+void vibe_sparkle() 
+{
+   if(is_button_pressed) 
+   {
+    for(int i = 0; i < NUM_LEDS; i++)
+    {
+	uint8_t bri = 255;
+		leds[i] = CHSV(global_hue, 200, bri);
+    addGlitter(10);
```

Comment:
In this code addGlitter is run once every pixel. Did you mean to call it outside the loop?

Reply:
disregard I see that add glitter only adds one potential glitter so needs to be called multiple times